.circleci/config.yml

```diff
@@ -548,11 +573,22 @@ workflows:
               only: /^v(\d+\.)(\d+\.)(\d+)(.*)?$/
             branches:
               only: unsigned
+      - build-ios-iphonesimulator:
+          context: mattermost-mobile-unsigned
+          requires:
+            - test
+          filters:
+            tags:
+              only: /^v(\d+\.)(\d+\.)(\d+)(.*)?$/
+            only:
+                - unsigned
+                
       - github-release:
           context: mattermost-mobile-unsigned
           requires:
             - build-android-unsigned
             - build-ios-unsigned
+            - build-ios-iphonesimulator
```

Comment:
Hope this is fine 'cause this will make easier for RainforestQA testing when referencing app from publicly accessible link.